safe_control_gym/experiments/base_experiment.py

```diff
@@ -409,10 +409,16 @@ def compute_metrics(self, data, verbose=False):
             'average_constraint_violation': np.asarray(self.get_episode_constraint_violation_steps()).mean(),
             'constraint_violation_std': np.asarray(self.get_episode_constraint_violation_steps()).std(),
             'constraint_violation': np.asarray(self.get_episode_constraint_violation_steps()) if len(self.get_episode_constraint_violation_steps()) > 1 else self.get_episode_constraint_violation_steps()[0],
+            'mean_t_wall_ms': np.asarray(self.get_t_wall()).mean()*1e3,
```

Comment:
Yes, thanks for pointing out. I propose a fix with a `try/except`.
Ultimately, i believe it would also be interesting to add the `t_wall` for the inference of neural networks.

Reply:
In another branch what we did is we measure the inference time in the `base_experiment.py` class itself, so we don't need to change every controller. I think this is a cleaner solution. For now I propose we remove this and instead update the base experiment later to measure the time

Reply:
Just a supplementary comment on the plan for the inference time metric. The inference time in the other branch is temporary. This metric should consider only the online optimization time and exclude all the variable logging etc. So in the end we will need to modify every controller to get this metric. 

I also agree on removing this first from `base_experiment.py` but keeping the timing in the `mpc_acados` as some initial progess for unifying this metric.

Reply:
For `linear_mpc`, if we add the option `'record_time': True` when `self.solver == 'qrqp'`, then we can get `t_wall_total` for that controller as well. I think its a really useful metric so maybe we can keep it in `mpc`, `linear_mpc`, and `mpc_acados`, but not in `base_experiment.py`. I think in the future we should look at how to get a good timing metric for all controllers though (that doesn't include logging time).